nornir/init_nornir.py

```diff
@@ -17,7 +18,18 @@ def load_inventory(
 ) -> Inventory:
     InventoryPluginRegister.auto_register()
     inventory_plugin = InventoryPluginRegister.get_plugin(config.inventory.plugin)
-    inv = inventory_plugin(**config.inventory.options).load()
+    inventory_plugin_params = config.inventory.options.copy()
+
+    init_params = inspect.signature(inventory_plugin).parameters
```

Comment:
This code is most likely not working yet. The idea would be to detect if the Inventory plugin allows passing down the configuration settings to the Host, Group and Defaults Inventory objects. This would allow us to avoid breaking existing inventory plugins.